model/src/main/kotlin/config/ScannerConfiguration.kt

```diff
@@ -30,6 +30,12 @@ import org.ossreviewtoolkit.utils.storage.FileStorage
  */
 @JsonIgnoreProperties(ignoreUnknown = true)
 data class ScannerConfiguration(
+    /**
+     * A flag to indicate whether packages that have a concluded license set (and non-empty authors) should be skipped
+     * in the scan.
+     */
+    val skipConcluded: Boolean = false,
```

Comment:
Can we document this option?
E.g. in https://github.com/oss-review-toolkit/ort/blob/master/model/src/main/resources/reference.conf

Reply:
I *listed* it in `reference.conf`, thanks for the reminder. That's not real "documentation", though.

Reply:
It would be more important to explain why authors are relevant here than in the code comment further down. Btw, I don't remember, was it already implemented that authors are associated as copyright holders to declared licenses?